### PR TITLE
docs: add section about Slack channel invite

### PR DESCRIPTION
**Pull request by timbru31**

### CODE REVIEW

**Comment by timbru31** [comment]
pinging @janpio because he is the GitHub issue template master ;)

File: .github/ISSUE_TEMPLATE/NEW_COMMITER_AND_PMC.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+name: 💌 Add a new commiter and PMC
+about: Steps to that need to be done in order to add a new commiter and PMC
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/NEW_COMMITER_AND_PMC.md]:
```suggestion
about: Steps that need to be done in order to add a new committer and PMC member
```

**Comment by janpio** [comment]
Filename: COMMITER => COMMITTER

You probably also need a normal template as fallback - not sure.

**Comment by timbru31** [comment]
> You probably also need a normal template as fallback - not sure.

Nope, GitHub let's you create new issues without templates, too (you can choose that when creating a new issue) :)

Screen Shot 2019-04-26 at 16 03 57
(other project but applies to every repo)

File: .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+---
+name: 💌 Add a new commiter and PMC
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md]:
```suggestion
name: 💌 Add a new committer and PMC member
```

File: .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md [hunk 3]
```diff
@@ -12,5 +18,6 @@ See the [README](../README.md) for details on each of the steps.
 - [ ] 6. Once 72 hours has elapsed from the board@ notice, add the committer to the PMC committee-info.txt file and LDAP (PMC Chair)[[details]](https://github.com/apache/cordova-new-committer-and-pmc#6-once-72-hours-has-elapsed-from-the-board-notice-add-the-committer-to-the-pmc-committee-infotxt-file-and-ldap-any-pmc-chair)
 - [ ] 7. Add the PMC member to the private@ list [[details]](https://github.com/apache/cordova-new-committer-and-pmc#7-automatically-subscribe-the-new-pmc-member-to-the-private-list)
 - [ ] 8. Add the PMC member to the JIRA 'PMC' User Role [[details]](https://github.com/apache/cordova-new-committer-and-pmc#8-add-the-pmc-member-to-the-jira-pmc-user-role)
-- [ ] 9. Ask the PMC member to Link their Github account [[details]](https://github.com/apache/cordova-new-committer-and-pmc#9-ask-the-pmc-member-to-link-their-github-account)
+- [ ] 9. Ask the PMC member to Link their GitHub account [[details]](https://github.com/apache/cordova-new-committer-and-pmc#9-ask-the-pmc-member-to-link-their-github-account)
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md]:
```suggestion
- [ ] 9. Ask the PMC member to link their GitHub account [[details]](https://github.com/apache/cordova-new-committer-and-pmc#9-ask-the-pmc-member-to-link-their-github-account)
```

File: .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md [hunk 4]
```diff
@@ -12,5 +18,6 @@ See the [README](../README.md) for details on each of the steps.
...
 - [ ] 7. Add the PMC member to the private@ list [[details]](https://github.com/apache/cordova-new-committer-and-pmc#7-automatically-subscribe-the-new-pmc-member-to-the-private-list)
 - [ ] 8. Add the PMC member to the JIRA 'PMC' User Role [[details]](https://github.com/apache/cordova-new-committer-and-pmc#8-add-the-pmc-member-to-the-jira-pmc-user-role)
-- [ ] 9. Ask the PMC member to Link their Github account [[details]](https://github.com/apache/cordova-new-committer-and-pmc#9-ask-the-pmc-member-to-link-their-github-account)
+- [ ] 9. Ask the PMC member to Link their GitHub account [[details]](https://github.com/apache/cordova-new-committer-and-pmc#9-ask-the-pmc-member-to-link-their-github-account)
+- [ ] 10. Invite the PMC member to the 'pmc' Slack channel [[details]](https://github.com/apache/cordova-new-committer-and-pmc#10-invite-the-pmc-member-to-the-pmc-slack-channel)
```
Comment (janpio) [inline .github/ISSUE_TEMPLATE/NEW_COMMITTER_AND_PMC.md]:
```suggestion
- [ ] 10. Invite the PMC member to the `#pmc` Slack channel [[details]](https://github.com/apache/cordova-new-committer-and-pmc#10-invite-the-pmc-member-to-the-pmc-slack-channel)
```

File: README.md [hunk 5]
```diff
@@ -145,6 +145,10 @@ Use this [CodePen](https://codepen.io/shazron/pen/yXVbXr) to encode it.
 3. Select `Add Users to a Role`
 4. Search for the user, and add to the `PMC` role
 
-## 9. Ask the PMC member to Link their Github account
+## 9. Ask the PMC member to Link their GitHub account
 
 Go to [Gitbox](https://gitbox.apache.org/setup/)
+
+## 10. Invite the PMC member to the 'pmc' Slack Channel
```
Comment (janpio) [inline README.md]:
```suggestion
## 10. Invite the PMC member to the `#pmc` Slack Channel
```

File: README.md [hunk 6]
```diff
@@ -145,6 +145,10 @@ Use this [CodePen](https://codepen.io/shazron/pen/yXVbXr) to encode it.
...
+
+If the new PMC member is using Slack, invite them to the private 'pmc' Slack channel
```
Comment (janpio) [inline README.md]:
```suggestion
If the new PMC member is using Slack, invite them to the private `#pmc` Slack channel
```

**Review by janpio — APPROVED** [review]

**Pull request merged**